allensdk/brain_observatory/demixer.py

```diff
@@ -33,23 +33,100 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 #
+from typing import Tuple, Optional
+import os
+import logging
+
+import numpy as np
 import scipy.sparse as sparse
 import scipy.linalg as linalg
-import numpy as np
-import os
 import matplotlib.pyplot as plt
-import logging
 import matplotlib.colors as colors
+
+import allensdk.internal.brain_observatory.mask_set as mask_set
 from allensdk.config.manifest import Manifest
 
-def demix_time_dep_masks(raw_traces, stack, masks):
-    '''
 
-    :param raw_traces: extracted traces
-    :param stack: movie (same length as traces)
-    :param masks: binary roi masks
-    :return: demixed traces
-    '''
+def identify_valid_masks(mask_array):
```

Comment:
Maybe add information about the type/shape of `mask_array`?

Reply:
I copy-pasted this as-is from the internal demixing module to not break any dependencies. I can try to figure it out...

Reply:
This functionality is undocumented and not very clear. I will make a note that it needs documentation but I don't want to hold up this PR for this function.